test/addons/argocd/test

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+
+from drenv import kubectl
+from drenv import commands
+
+
+def wait(cluster):
+    print("Waiting application to be healthy")
+    kubectl.wait(
+        "application",
+        "guestbook",
+        "--for=jsonpath={.status.health.status}=Healthy",
+        "--namespace=argocd",
+        "--timeout=300s",
```

Comment:
There is no need to wait 300 seconds for deploying the guestbook. The entire test takes here about 12 seconds.

We need a reasonable timeout that will not fail on the CI (with flaky network), but will fail fast enough so we don't waste time debugging this code. I would use 60 seconds since this seems to take only few seconds.

If we see failures in the CI we can double the timeout, but this likely means we need to deploy a smaller app from local sources instead of github. Accessing  resources in github from the CI lab is flaky - this is why we cache all resources.

Reply:
ok. updated to 60s.